pgcontents/checkpoints.py

```diff
@@ -65,7 +65,7 @@ def rename_checkpoint(self, checkpoint_id, old_path, new_path):
         """Rename a checkpoint from old_path to new_path."""
         with self.engine.begin() as db:
             return move_single_remote_checkpoint(
-                db, old_path, new_path, checkpoint_id,
+                db, self.user_id, old_path, new_path, checkpoint_id,
```

Comment:
I see tests for renaming directories, but I couldn't find tests on how checkpoints interact with things:
https://github.com/quantopian/pgcontents/blob/master/pgcontents/tests/test_pgmanager.py#L124

I'm not quite sure what the test logic would be either & tried to find what actually calls rename_checkpoint and couldn't figure out where it would be:
https://github.com/jupyter/notebook/search?utf8=%E2%9C%93&q=rename_checkpoint